lms/djangoapps/ora_staff_grader/tests/test_serializers.py

```diff
@@ -239,6 +239,8 @@ class TestSubmissionMetadataSerializer(TestCase):
             "submissionUuid": "a",
             "username": "foo",
             "teamName": "",
+            'email': None,
+            'fullname': None,
```

Comment:
why adding this if the fields can be None? can we add that they can be empty too? So this still works as before.